src/main/java/de/tum/in/www1/artemis/repository/UserRepository.java

```diff
@@ -113,8 +118,8 @@ public interface UserRepository extends JpaRepository<User, Long>, JpaSpecificat
      * @return list of found users that match the search criteria
      */
     @EntityGraph(type = LOAD, attributePaths = { "groups" })
-    @Query("select user from User user where :#{#groupName} member of user.groups and "
-            + "(user.login like :#{#loginOrName}% or concat_ws(' ', user.firstName, user.lastName) like %:#{#loginOrName}%)")
+    @Query("select user from User user where user.isDeleted = false and " + "(:#{#groupName} member of user.groups and "
```

Comment:
```suggestion
    @Query("select user from User user where user.isDeleted = false and (:#{#groupName} member of user.groups and "
```

Reply:
Maybe even think about rewriting these queries properly with capitalization and Textblocks